### ISSUE TITLE
[Feature Request] Prop to remove border for v-data-table row

### ISSUE THREAD

**Issue by holuborhee** [vuetifyjs/vuetify#14412]
### Problem to solve
When using a v-data-table to render data, it comes with a default border at the top and bottom of each row. There should be an easy way for users to do away with these borders.

### Proposed solution
Create a prop `no-row-border` for users to remove the border willingly. The prop should not remove border from the footer and header, there should another prop to achieve that, preferably `no-header-border` and `no-footer-border`

**Comment by adil-portable** [comment]
@johnleider @kaelwd 
Hi, My name is Muhammad Adil Khan. I work for Portable, Australia and our goal is to contribute to open source projects on Github. My team and I would like to contribute to this project. Please let us know if we work on this change request and would like to work on other tickets as well.

I look forward to hearing from you soon.

Thanks,
Adil

**Comment by adil-portable** [comment]
@holuborhee I looked into the code and its more like CSS thing. We normally override the existing CSS with custom CSS to get the require result.

**Comment by holuborhee** [comment]
> @holuborhee I looked into the code and its more like CSS thing. We normally override the existing CSS with custom CSS to get the require result.

Yeah, it is possible to use custom CSS, and I've always resolved to use that. However, I still think having a prop for this common behavior makes for easier customization and a better experience for the developer.

**Comment by johnleider** [comment]
@nekosaur thoughts on this? https://github.com/vuetifyjs/vuetify/issues/14412#issuecomment-984297921

**Comment by adil-portable** [comment]
@holuborhee

**Comment by adil-portable** [comment]
@holuborhee
![image](https://user-images.githubusercontent.com/94330648/145509910-c48e8761-a101-4483-bac0-089e86a2ddfa.png)
![image](https://user-images.githubusercontent.com/94330648/145510934-60aa9f8a-c42b-4409-9507-0a39372dc198.png)
![image](https://user-images.githubusercontent.com/94330648/145511045-e46de81a-cf33-4801-8c81-babaf1886e8f.png)
![image](https://user-images.githubusercontent.com/94330648/145511088-52ea2efe-0b34-4955-bf73-c9746ca4aba0.png)

**Comment by adil-portable** [comment]
@johnleider @KaelWD @nekosaur It has been 2 week since I submitted the PR. Could you please review it? 
Thanks and best regards,
Adil

**Comment by KaelWD** [comment]
@adil-portable 
- PRs have to be submitted to this repo, not your fork
- We don't recommend working on feature requests that haven't been accepted yet as it may not be merged

**Comment by adil-portable** [comment]
@KaelWD A bit confused on how to submit to this repo. 
![image](https://user-images.githubusercontent.com/94330648/148879809-d61bfe99-de5d-4e2f-975a-b818a6563aa9.png)
Should I select master branch? Could you please guide me? thanks

**Comment by holuborhee** [comment]
> @KaelWD A bit confused on how to submit to this repo. ![image](https://user-images.githubusercontent.com/94330648/148879809-d61bfe99-de5d-4e2f-975a-b818a6563aa9.png) Should I select master branch? Could you please guide me? thanks

@adil-portable 
This link should help https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork.

Also, I think working on this issue (feature request) is not encouraged because it has not been accepted.

**Comment by adil-portable** [comment]
@holuborhee  thanks
@KaelWD I have pointed the branch to this repo https://github.com/vuetifyjs/vuetify/pull/14593 
please review the PR. As you have mentioned "We don't recommend working on feature requests that haven't been accepted yet as it may not be merged". If you think the work I have done is good to go please merge or update me if you think the requested feature is not required. 

I look forward to hearing from you soon.

Thanks

**Comment by alniv94** [comment]
any update on this?
